<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="previewWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSplitter" name="streamSplitter">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="opaqueResize">
          <bool>false</bool>
         </property>
         <property name="handleWidth">
          <number>11</number>
         </property>
         <widget class="QGroupBox" name="boxColorStream">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>320</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Color Stream</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="boxDepthStream">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>320</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Depth Stream</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="boxSkeletonStream">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>320</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Skeleton Stream</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="icon">
        <iconset>
         <normaloff>../../../../../../.designer/backup/icons/record.png</normaloff>../../../../../../.designer/backup/icons/record.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Record</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="title">
           <string>Capture Streams</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="1">
            <widget class="QLineEdit" name="leDepthOut">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="leColorOut">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="leSkeletonOut">
             <property name="dragEnabled">
              <bool>false</bool>
             </property>
             <property name="clearButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="cbCaptureDepth">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Depth</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="cbCaptureSkeleton">
             <property name="text">
              <string>Skeleton</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="cbCaptureColor">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Color</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pbFileColor">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pbFileDepth">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="pbFileSkeleton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pbRecord">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Record</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../../../../.designer/backup/icons/record.png</normaloff>../../../../../../.designer/backup/icons/record.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Scene Config</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="1">
            <widget class="QPushButton" name="pbGenerateConfig">
             <property name="text">
              <string>Generate Config</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="leConnectionId"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Kinect Connection Id:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Log Level</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QComboBox" name="cbLogLevel">
                <property name="currentIndex">
                 <number>3</number>
                </property>
                <property name="modelColumn">
                 <number>0</number>
                </property>
                <item>
                 <property name="text">
                  <string>trace - 0</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>debug - 1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>info - 2</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>warn - 3</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>err - 4</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>critical - 5</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>off - 6</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Play</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="0">
         <widget class="QGroupBox" name="gbFrames">
          <property name="title">
           <string>Frames</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QSlider" name="hsFrames">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="labelFramesFrom">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelFramesUntil">
               <property name="text">
                <string>-1</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pbPlaySkeleton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Play</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="leSkeletonIn">
            <property name="text">
             <string>Skeleton.kif</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbPlaySkeletonPath">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pbLoad">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cbCaptureDepth</sender>
   <signal>toggled(bool)</signal>
   <receiver>leDepthOut</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbCaptureSkeleton</sender>
   <signal>toggled(bool)</signal>
   <receiver>leSkeletonOut</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>528</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbCaptureSkeleton</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbFileSkeleton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbCaptureDepth</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbFileDepth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbCaptureColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>leColorOut</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbCaptureColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbFileColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>469</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
